diff -Nru j2sdk.orig/j2se/src/solaris/back/exec_md.c j2sdk/j2se/src/solaris/back/exec_md.c
--- j2sdk.orig/j2se/src/solaris/back/exec_md.c	Fri Sep  6 09:29:12 2002
+++ j2sdk/j2se/src/solaris/back/exec_md.c	Wed Mar 26 15:40:26 2003
@@ -72,7 +72,7 @@
     }
     argv[i] = NULL;  /* NULL terminate */
 
-    if ((pid = fork1()) == 0) {
+    if ((pid = fork()) == 0) {
         /* Child process */
         int i, max_fd;
 
