diff -Nru j2sdk-1.4.1.orig/control/make/Makefile j2sdk-1.4.1/control/make/Makefile
--- j2sdk-1.4.1.orig/control/make/Makefile	Fri Sep  6 09:00:15 2002
+++ j2sdk-1.4.1/control/make/Makefile	Fri Mar 21 11:24:02 2003
@@ -7,6 +7,8 @@
 # @(#)Makefile	1.250 02/06/24
 #
 
+SHELL=/bin/bash
+
 BUILDDIR=.
 include $(BUILDDIR)/common/Platform.gmk
 include $(BUILDDIR)/common/Defs.gmk
diff -Nru j2sdk-1.4.1.orig/j2se/make/Makefile j2sdk-1.4.1/j2se/make/Makefile
--- j2sdk-1.4.1.orig/j2se/make/Makefile	Fri Sep  6 09:07:02 2002
+++ j2sdk-1.4.1/j2se/make/Makefile	Fri Mar 21 11:43:27 2003
@@ -29,6 +29,8 @@
 #    plugin-clobber -- clobber the plugin build (defined in Plugin.gmk) (32-bit only)
 #   
 
+SHELL=/bin/bash
+
 all:: 
 	@$(ECHO) $(PLATFORM) $(ARCH) $(RELEASE) build started: $(shell $(DATE) '+%y-%m-%d %H:%M')
 
