diff -Nru j2sdk.orig/j2se/make/java/net/Makefile j2sdk/j2se/make/java/net/Makefile
--- j2sdk.orig/j2se/make/java/net/Makefile	Fri Sep  6 09:07:34 2002
+++ j2sdk/j2se/make/java/net/Makefile	Tue Mar 25 19:42:05 2003
@@ -70,7 +70,7 @@
   COMPILER_WARNING_LEVEL=2
 else
   LIBSOCKET = -lsocket
-  OTHER_LDLIBS = $(LIBSOCKET) -lnsl -ldl $(JVMLIB)
+  OTHER_LDLIBS = $(LIBSOCKET) -lnsl -ldl $(JVMLIB) -lpthread
 endif
 
 CLASSES.export += java.lang.Integer java.io.FileDescriptor java.net.InetAddressImplFactory java.net.Inet4AddressImpl java.net.Inet6AddressImpl
diff -Nru j2sdk.orig/j2se/make/java/nio/Makefile j2sdk/j2se/make/java/nio/Makefile
--- j2sdk.orig/j2se/make/java/nio/Makefile	Fri Sep  6 09:07:34 2002
+++ j2sdk/j2se/make/java/nio/Makefile	Tue Mar 25 22:41:44 2003
@@ -112,7 +112,7 @@
 endif
 ifeq ($(PLATFORM), linux)
 OTHER_CFLAGS += -Werror
-OTHER_LDLIBS += -L$(LIBDIR)/$(LIBARCH)$(SUFFIX) -ljava$(SUFFIX) -lnet$(SUFFIX)
+OTHER_LDLIBS += -L$(LIBDIR)/$(LIBARCH)$(SUFFIX) -ljava$(SUFFIX) -lnet$(SUFFIX) -lpthread
 endif
 ifeq ($(PLATFORM), solaris)
 OTHER_LDLIBS += $(JVMLIB) -lsocket -lposix4 \
